terraform/modules/happy-service-eks/variables.tf

```diff
@@ -306,8 +306,10 @@ variable "routing" {
 
 variable "sidecars" {
   type = map(object({
-    image : string
-    tag : string
+    image : string,
+    tag : string,
+    cmd : optional(list(string), []),
+    args : optional(list(string), []),
     port : optional(number, 80)
     scheme : optional(string, "HTTP")
     memory : optional(string, "100Mi")
```

Comment:
Nit: it would be nice to have consistent trailing commas or no trailing commas in this map